src/main/resources/frontend/sentiment-frontend.html

```diff
@@ -98,29 +98,39 @@ <h4>Anzahl offensive vs. non-offensive Tweets</h4>
 					<div class="content p-2">
 						<h4>Verlauf offensive & non-offensive Tweets</h4>
 						<v-date-picker @input="updateLineChart()" popover-align="center" :max-date='new Date()' mode='range' v-model="selectedDate">
-						</v-date-picker>	
+						</v-date-picker>
 						<div>
 							<canvas id="lineChart"></canvas>
-						</div>		
+						</div>
 					</div>
 				</div>
 			</div>
 			<hr>
 			<div class="row my-3">
 				<div class="col-sm-6">
 					<div class="content p-2">
-							<h4>Zufälliger als offensive eingestufter Tweet:</h4>		
+							<h4>Zufälliger als offensive eingestufter Tweet:</h4>
 							<div id=offTweet></div>
+							<div class="border-box">
+								<script type="text/javascript">
```

Comment:
Bitte kein inline-scripting! ;)

Reply:
Ähm also nur simples inline scripting und sonstige Funktionen in die vue.app... keine <script>-tags mit js drin.